securesystemslib/rsa_keys.py

```diff
@@ -334,12 +361,22 @@ def create_rsa_signature(private_key, data, scheme='rsassa-pss-sha256'):
         'pyca_crypto')
 
     if scheme.startswith('rsassa-pss'):
+      if salt_length == SaltLength.PSSSaltLengthAuto:
+        salt_length = padding.PSS.MAX_LENGTH
+      elif salt_length == SaltLength.PSSSaltLengthEqualsHash:
+        salt_length = digest_obj.algorithm.digest_size
+      else:
+        raise securesystemslib.exceptions.UnsupportedAlgorithmError('Unsupported'
+          ' salt length is specified: ' + repr(salt_length))
```

Comment:
Can we have different variable names for the passed salt_length (of type SaltLength) and the salt_length we are passing to `sign()` (presumably an int?)? Reusing variable names can be tricky to read and lead to unexpected errors later.